docker.Makefile

```diff
@@ -26,23 +24,15 @@ ENVVARS = -e VERSION=$(VERSION) -e GITCOMMIT -e PLATFORM -e TESTFLAGS -e TESTDIR
 # Some Dockerfiles use features that are only supported with BuildKit enabled
 export DOCKER_BUILDKIT=1
 
+# Progress plain to show container logs with buildx/bake
+export BUILDKIT_PROGRESS=plain
+
```

Comment:
for local work, it's fine to use the default output (less noisy), as long as CI uses `plain` (I'm guessing it already does if there's no TTY attached)